examples/06_material/resources/config.ron

```diff
@@ -0,0 +1,10 @@
+(
+  dimensions: None,
```

Comment:
Let's use recommended formatting (so 4 spaces).

Reply:
I've copied it from other examples. Should I fix formatting there too?